pkg/apis/certmanager/validation/issuer.go

```diff
@@ -184,6 +184,17 @@ func ValidateACMEIssuerDNS01Config(iss *v1alpha1.ACMEIssuerDNS01Config, fldPath
 				}
 			}
 		}
+		if p.DNSMadeEasy != nil {
+			if numProviders > 0 {
+				el = append(el, field.Forbidden(fldPath.Child("dnsmadeeasy"), "may not specify more than one provider type"))
+			} else {
+				numProviders++
+				el = append(el, ValidateSecretKeySelector(&p.DNSMadeEasy.SecretKey, fldPath.Child("dnsmadeeasy", "secretKeySecretRef"))...)
+				if len(p.DNSMadeEasy.APIKey) == 0 {
+					el = append(el, field.Required(fldPath.Child("dnsmadeeasy", "apiKey"), ""))
```

Comment:
Is `baseURL` not required?

Reply:
It's not, no. I'll address your comments in pkg/apis/certmanager/v1alpha1/types_issuer.go to make that easier to recognize.